README.md

```diff
@@ -33,7 +33,7 @@ When running with `parallel` set to true, the final reports can be merged by usi
 
 ## Configuration
 
-Configuration is optional. It should be put in a file at `config/coverage.js`. 
+Configuration is optional. It should be put in a file at named `coverage.js` inside your ember app config folder (defaults to `config/`).
```

Comment:
Thanks for taking a look at this! I've reworded, let me know if the new message is more inline with what you have in mind.